docs/src/modules/components/DemoToolbar.js

```diff
@@ -562,56 +555,32 @@ export default function DemoToolbar(props) {
           </Box>
         </Fade>
         <Box sx={{ ml: 'auto' }}>
-          <ToggleCodeTooltip
+          <Button
+            aria-controls={openDemoSource ? demoSourceId : null}
+            data-ga-event-category="demo"
+            data-ga-event-label={demo.gaLabel}
+            data-ga-event-action="expand"
+            onClick={handleCodeOpenClick}
```

Comment:
You can delete the definition of `ToggleCodeTooltip` if not used anymore

Reply:
Good call! 🤙